.github/workflows/pr-labels.yml

```
@@ -0,0 +1,160 @@
+name: PR Title Check and Add Label
+
+on:
+  pull_request:
+    types: [ opened, edited, synchronize ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)